packages/prepo-shared-contracts/test/fixtures/FixedUintValueFixture.ts

```diff
@@ -0,0 +1,7 @@
+import { ethers } from 'hardhat'
+import { FixedUintValue } from '../../types/generated'
+
+export async function fixedUintValueFixture(): Promise<FixedUintValue> {
+  const Factory = await ethers.getContractFactory('FixedUintValue')
+  return (await Factory.deploy()) as FixedUintValue
```

Comment:
```suggestion
  const factory = await ethers.getContractFactory('FixedUintValue')
  return (await factory.deploy()) as FixedUintValue
```
I know else where this has been merged this way, but been meaning to switch them over to just camelcase, or lowercase in this case.